framework/sessions/__init__.py

```diff
@@ -85,12 +85,17 @@ def prepare_private_key():
 #     return user_session
 
 def get_session():
-    session_key = request.cookies.get(settings.COOKIE_NAME)
-    if session_store.exists(session_key=session_key):
-        user_session = session_store.get(session_key)
-    else:
-        user_session = session_store.create()
-    return user_session
+    secret = settings.SECRET_KEY
```

Comment:
A follow-up comment: here `settings` is [the correct one](https://github.com/CenterForOpenScience/osf.io/blob/b5c28d2b096c0090fc34f7f2ad151fb9d546e333/framework/sessions/__init__.py#L18)